### PR TITLE
[fix] SQLHelpers scope

### PR DESCRIPTION
**Pull request by sanderpick**
## Summary

Let's devs use the SQLHelpers lib without first deploying it as a standalone contract.

## Details

The lib had all methods public, which means to use it a dev has to actually deploy the contract. This is cumbersome. I've changed all methods to `internal`, which means they will simply be included with the contract that is using them. This is how OZ Strings / Math, etc. libs are written.

## How it was tested

Updated unit tests use a wrapper library with public methods.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "./contracts/": "./contracts/"
   },
   "scripts": {
-    "build": "npx tsc -p ./tsconfig.build.json",
-    "clean": "rm -rf artifacts && rm -rf typechain-types && rm -rf cache && rm -rf coverage && rm -f coverage.json",
+    "build": "hardhat compile && npx tsc -p ./tsconfig.build.json",
```
Comment (joewagner) [inline package.json:22]: nice

**Review by joewagner — APPROVED** [review]
This is much better, do we need to bump a major version for this since it changes how deployment works?

**Comment by carsonfarmer** [comment]
Nice, this is great! Once we merge this one, we can use the new release flow to pop out a release.

```
npm version major
git push --follow-tags
```

**Comment by joewagner** [comment]
> > This is much better, do we need to bump a major version for this since it changes how deployment works?
> 
> do you mean how the contract is deployed?

Yeah.  Say someone has built something that deploys @tableland/evm/contracts/utils/SQLHelpers.sol and then calls the resulting public functions.  If we publish this to npm and that person runs their deploy again it wouldn't work as expected right?

**Pull request merged**